bindings/pyroot/pythonizations/test/CMakeLists.txt

```diff
@@ -127,6 +127,13 @@ endif()
 
 # RTensor pythonizations
 if (dataframe AND tmva)
+    if(NOT MSVC OR CMAKE_SIZEOF_VOID_P EQUAL 4 OR win_broken_tests)
```

Comment:
This test will also need a check that the Python version is at least 3.X

Reply:
made the change, thanks!